dashboard-ui/src/components/SurveyResults/resultsTable.jsx

```diff
@@ -69,19 +70,13 @@ const STARTING_HEADERS = [
     "VR Experience Level",
     "VR Comfort Level",
     "Additional Information About Discomfort",
-    "Note Page - Time Taken (mm:ss)"
+    "Note Page - Time Taken (Minutes)"
 ];
 
-function formatTime(seconds, includeHours = false) {
-    seconds = Math.round(seconds);
-    let hours = (Math.floor(seconds / 60 / 60) % 60);
-    hours = hours.toString().length < 2 ? '0' + hours.toString() : hours.toString();
-    let minutes = Math.floor(seconds / 60);
-    minutes = includeHours ? minutes % 60 : minutes;
-    minutes = minutes.toString().length < 2 ? '0' + minutes.toString() : minutes.toString();
-    let formatted_seconds = seconds % 60;
-    formatted_seconds = formatted_seconds.toString().length < 2 ? '0' + formatted_seconds.toString() : formatted_seconds.toString();
-    return includeHours ? `${hours}:${minutes}:${formatted_seconds}` : `${minutes}:${formatted_seconds}`;
```

Comment:
I feel like we shouldn't just delete the old code entirely.  Could we have a `formatTimeInMinutes` and `formatTimeInMMSS` or something like that?